bazel/repositories.bzl

```diff
@@ -92,7 +92,7 @@ def _local_repo_impl(name, **kwargs):
 def _git_repo(name, **kwargs):
     _git_repo_impl(name, **kwargs)
 
-def _local_repo(name, **kwargs):
+def _local_repo(name, **kwargs):  # buildifier: disable=unused-variable
```

Comment:
This function was causing the run-container-lint check to fail due to it being unused. It seems that its purpose was to develop bcc/bpftrace locally and the code to do so was commented in the file (line 172,173)